scripts/upload-data.js

```diff
@@ -139,10 +140,10 @@ const main = async () => {
           name: "cluster_id",
           type: "number",
         },
-        {
-          name: "name",
-          type: "string",
-        },
+        // {
+        //   name: "name",
+        //   type: "string",
+        // },
```

Comment:
This is temporary right? We'll want to enable this back in once we get the data?